test/functional/issue_templates_controller_test.rb

```diff
@@ -269,12 +273,6 @@ def child_project_setup
     @request.session[:user_id] = 1
   end
 
-  def checklist_plugin_enabled
-    Redmine::Plugin.register(:redmine_checklists) do
```

Comment:
When use Redmine::Plugin.register, actual directory is also required.
So, change to use stub instead of plugin register.
Ref. https://www.redmine.org/projects/redmine/repository/revisions/18064/diff/trunk/lib/redmine/plugin.rb